runner/run.go

```diff
@@ -148,9 +148,9 @@ func RunStage(runContext *TestRunContext, ftwCheck *check.FTWCheck, testCase sch
 		return err
 	}
 
-	// Do not even run test if result is overridden. Just use the override and display the overridden result.
+	// Do not even run test if result is overridden. Directly set and display the overridden result.
 	if overridden := overriddenTestResult(ftwCheck, &testCase); overridden != Failed {
-		runContext.Stats.addResultToStats(overridden, &testCase)
+		runContext.Result = overridden
```

Comment:
This is the one line fix. The result is not added here to the stats, but is updated with the overridden result. The caller will then add it to the stats like any other executed test.